src/lib/plugins/blocks.js

```diff
@@ -4,12 +4,7 @@ module.exports.player=function(player,serv)
 {
   player.changeBlock=async (position,blockType,blockData) =>
   {
-    serv.players
-      .filter(p => p.world==player.world && player!=p)
```

Comment:
when you dig or place a block you don't need to send that info to the vanilla client, it already knows